src/OrchardCore.Modules/OrchardCore.Users/TimeZone/Services/UserTimeZoneService.cs

```diff
@@ -95,15 +94,6 @@ private async ValueTask<string> GetTimeZoneIdAsync(string userName)
         return timeZoneId;
     }
 
-    public override Task DeletedAsync(UserDeleteContext context)
-        => ForgetCacheAsync(context.User.UserName);
-
-    public override Task UpdatedAsync(UserUpdateContext context)
-        => ForgetCacheAsync(context.User.UserName);
-
-    public override Task DisabledAsync(UserContext context)
-        => ForgetCacheAsync(context.User.UserName);
-
     private Task ForgetCacheAsync(string userName)
```

Comment:
This is now not used anymore, right?

Reply:
This is still needed to clear the cache when the obsolete method `UpdateAsync` is called. It will no longer be needed in 3.0 when we remove the 2 obsolete methods on the `IUserTimeZoneService` interface.